astropy/units/astrophys.py

```diff
@@ -14,7 +15,6 @@
 # To ensure si units of the constants can be interpreted.
 set_enabled_units([si])
 
-import numpy as _np  # noqa: E402
```

Comment:
@mhvk , are you sure the numpy import wasn't down here for a specific reason?

Reply:
If it was to keep numpy private here that would be resolved by https://github.com/astropy/astropy/pull/15899#pullrequestreview-1827893978.

Reply:
It was originally there to ensure it did not pollute the namespace. But with #15862, we define an `__all__`, so we can do whatever we want. Indeed, arguably we should have changed this in #15862!